aztec/src/main/kotlin/org/wordpress/aztec/AztecText.kt

```diff
@@ -406,6 +406,8 @@ class AztecText : EditText, TextWatcher {
     }
 
     fun fromHtml(source: String) {
+        editableText.clear()
```

Comment:
This creates a phantom empty text history point (edit text, switch to HTML, switch back, tap redo and the text is gone). Adding `disableTextChangedListener()` should help though.